manager/integration/tests/test_basic.py

```diff
@@ -519,6 +502,115 @@ def backupstore_test(client, host_id, volname, size):
     assert not found
 
 
+@pytest.mark.coretest   # NOQA
+def test_listing_backup_volume(clients, base_image=""):   # NOQA
+    for host_id, client in clients.iteritems():
+        break
+    lht_hostId = get_self_host_id()
+
+    # create 2 volumes.
```

Comment:
Create 3 with kind of random names, `volume-test-<random chars>`. We don't know the exact order which volume would be read first, which one second. But we want to make sure after reading an error volume, the read will continue.

Using random names ensures that at least in the multiple runs, the error volume will not always be the last one to be read.